### ISSUE TITLE
False Positive/Negative: Possible response check problem in Shellshock plugin

### ISSUE THREAD

**Issue by iasdeoupxe** [sullo/nikto#749]
A few years ago d051c3e4db1779779dd5d02c6a9a42242a09e1cc had changed multiple occurrences of the following string in [nikto_shellshock.plugin](https://github.com/sullo/nikto/blob/master/program/plugins/nikto_shellshock.plugin):

```
Nikto-Added-CVE-2014-6271
Nikto-Added-CVE
```

to:

```
93e4r0-CVE-2014-6278
93e4r0-CVE
```

There are still a few of these left:

```
nikto-added-cve-2014-6271
```

Shouldn't these be changed to this as well?

```
93e4r0-cve-2014-6278
```

**Comment by sullo** [comment]
Fixed in 2.5.0 breach. Release imminent.

**Comment by iasdeoupxe** [comment]
Additional fix was required: #752
